META

```diff
@@ -0,0 +1,8 @@
+# Specification for the "ocamlbuild" library
+requires = "unix"
+version = "[distributed with Ocaml]"
```

Comment:
I think we should rather use our own version numbers, the current choice seems to be 0.9, according to #3 and #5 .